.gitignore

```diff
@@ -17,6 +17,7 @@ lerna-debug.log*
 
 dist
 *.local
+**/.yextrc
```

Comment:
This shouldn't be ignored. The original purpose is for ProServe who often swap between repos often and need to re-init. This makes it easy so they don't need to figure out which business it's for and it auto-inits when they start dev'ing.

Reply:
I'm a little confused, is ProServe helping to develop yext/pages? This gitignore only affects development within this repo right?

Reply:
This gitignore doesn't somehow affect https://github.com/yext/pages-starter-react-locations does it?

Reply:
Yea that's true. Though that file shouldn't even exist in this repo.

Reply:
Oh it's from when I was testing in the playground folder.